### PR TITLE
(fix) Standardize home page dashboard links

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

This PR standardizes how we add dashboards to the home page.

### CODE REVIEW

File: packages/esm-appointments-app/src/routes.json [hunk 1]
```diff
@@ -30,16 +35,10 @@
...
-    },
     {
       "name": "todays-appointments-dashboard",
       "slot": "todays-appointment-slot",
       "component": "homeAppointments"
     }
-  ],
-  "pages": []
+  ]
 }
```
Comment (ibacher) [inline packages/esm-appointments-app/src/routes.json:44]:
```suggestion
}

```

**Review by ibacher — APPROVED** [review]

**Review by vasharma05 — DISMISSED** [review]
LGTM!
Thanks!

File: packages/esm-patient-search-app/src/root.component.tsx [hunk 2]
```diff
@@ -4,9 +4,9 @@ import PatientSearchPageComponent from './patient-search-page/patient-search-pag
 
 const PatientSearchRootComponent: React.FC = () => {
   return (
-    <BrowserRouter basename={`${window['getOpenmrsSpaBase']()}search`}>
+    <BrowserRouter basename={window.getOpenmrsSpaBase()}>
       <Routes>
-        <Route path="" element={<PatientSearchPageComponent />} />
+        <Route path="/search" element={<PatientSearchPageComponent />} />
```
Comment (ibacher) [inline packages/esm-patient-search-app/src/root.component.tsx:9]: Shouldn't this be `"search"`? (`getOpenmrsSpaBase()` returns `/openmrs/spa/`)

**Review by ibacher — APPROVED** [review]
LGTM assuming the e2e tests pass.

File: packages/esm-patient-search-app/src/root.component.tsx (hunk 2 above)
Comment (denniskigen) [reply packages/esm-patient-search-app/src/root.component.tsx:9]: It should! Thanks

**Pull request merged**